client/verta/verta/endpoint/_endpoint.py

```diff
@@ -403,6 +405,35 @@ def get_status(self):
 
         return response_json
 
+    def get_logs(self):
```

Comment:
@convoliution how big do these logs get? Is there a chance we're going to pull down a 1000 line error and put it into a `RuntimeError`? Also, dare I ask what happens when `get_logs` hits an error?

Reply:
> how big do these logs get? Is there a chance we're going to pull down a 1000 line error and put it into a RuntimeError?

In theory, there's no upper bound on their size, since it's whatever logs are coming out of the model container.

In practice, in the event of a deployment failure, it's 5 lines of Flask startup logs + any startup logs for external libraries (TensorFlow tends to have a lot) + the offending exception's stack trace. So 10–30 lines perhaps?

The previous `status_dict['components'][0].get('message', "no error message available")` relied on logic in the deployment backend to extract just the stack trace, but it's currently broken (hence this PR) and I'd prefer to not reinvent it here 😅 

> Also, dare I ask what happens when get_logs hits an error?

I suppose the user would see a rather unhelpful stack trace coming out of `Endpoint.get_logs()`. I'm not sure how many layers of robust exception handling we'd like.